presto-main/src/main/java/com/facebook/presto/catalogserver/CatalogServer.java

```diff
@@ -43,15 +43,15 @@
 {
     private static final String EMPTY_STRING = "";
 
-    private final Metadata metadataProvider;
+    private final Metadata metadata;
```

Comment:
Making this simple rename refactoring in a separate PR would make this PR easier to follow.

Reply:
This would lead to confusion, as we have new MetadatResolver now.